org.guitarix.Guitarix.json

```diff
@@ -0,0 +1,159 @@
+{
+    "app-id": "org.guitarix.Guitarix",
+    "branch": "20.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "guitarix",
+    "rename-desktop-file": "guitarix.desktop",
+    "rename-icon": "gx_head",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc",
+        "--socket=x11",
+        /* Note playback */
+        "--socket=pulseaudio",
+        /* MIDI */
+        "--device=all",
+        /* For Jack over pipewire */
+        "--socket=system-bus",
```

Comment:
Is this needed? On OBS Studio, there's no system-bus permission set and JACK still works

Reply:
As I remember it was needed but it is not impossible that this changed. I'll test again without.

Reply:
it is actually needed.